src/runtimes/node/in_source_config/index.ts

```diff
@@ -101,27 +112,28 @@ export const findISCDeclarations = (
   let scheduleFound = false
 
   const getAllBindings = createBindingsMethod(ast.body)
-  const { configExport, defaultExport, handlerExports } = getExports(ast.body, getAllBindings)
-  const isV2API = handlerExports.length === 0 && defaultExport !== undefined
+  const { configExport, handlerExports, hasDefaultExport, inputModuleFormat } = traverseNodes(ast.body, getAllBindings)
```

Comment:
We were not using the contents of `defaultExport` for anything, just checking whether it exists or not. So I've changed it to a boolean that is now `true` if ESM or CJS default exports are found.